helpdesk_mgmt_project/models/helpdesk_ticket.py

```diff
@@ -14,6 +16,34 @@ class HelpdeskTicket(models.Model):
         store=True,
     )
 
+    @api.model
+    def _default_company_id(self):
+        if self._context.get("default_project_id"):
+            return (
+                self.env["project.project"]
+                .browse(self._context["default_project_id"])
+                .company_id
+            )
+        return self.env.company
+
+    # Override field
+    company_id = fields.Many2one(
+        "res.company",
+        string="Company",
+        compute="_compute_company_id",
+        store=True,
+        readonly=False,
+        required=True,
+        copy=True,
+        default=_default_company_id,
+    )
+
+    @api.depends("project_id.company_id")
+    def _compute_company_id(self):
+        for ticket in self:
```

Comment:
a compute method MUST return a value for each element.
Your method doesn't respect this constraint. Please adapt